app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminClickLogoActionGroup.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminClickLogoActionGroup">
+        <click selector="{{AdminMenuSection.logo}}" stepKey="clickLogoInAdmin"/>
+        <seeInCurrentUrl url="{{AdminDashboardPage.url}}" stepKey="seeAdminDashboardUrl"/>
```

Comment:
Please remove this assertion. This Action Group is supposed only to click on the element.
```suggestion
        <waitForPageLoad stepKey="waitForAdminDashboardPageLoaded"/>
```

To check the Admin Dashboard, use the following Action Group `AssertAdminDashboardPageIsVisibleActionGroup`.